src/main/pages/che-7/overview/assembly_running-che-locally.adoc

```diff
@@ -78,7 +78,7 @@ include::proc_installing-che-on-codeready-containers-using-chectl.adoc[leveloffs
 
 include::proc_installing-che-on-kind-using-chectl.adoc[leveloffset=+1]
 
-== Deploying multi-user {prod-short} in multi-user mode
+== Deploying multiuser {prod-short} in multiuser mode
```

Comment:
Unfortunately, this heading has an auto-generated ID, so it will change and break links. Specifically, it is linked from the following two places:
```
$ grep -nR deploying-multi-user-che-in-multi-user-mode
src/main/pages/che-7/end-user-guide/assembly_configuring-openshift-oauth.adoc:19:* For {prod-short} deployed in link:{site-baseurl}/che-7/running-che-locally/#deploying-multi-user-che-in-multi-user-mode[multi-user mode]:
src/main/pages/che-7/end-user-guide/assembly_configuring-github-oauth.adoc:22:* For {prod-short} deployed in link:{site-baseurl}/che-7/running-che-locally/#deploying-multi-user-che-in-multi-user-mode[multi-user mode]:
```
Please, either update the links, or add an ID (an incorrect one, `deploying-multi-user-che-in-multi-user-mode`, just to ensure the links don't break).